source/common/json/json_streamer.h

```diff
@@ -51,24 +51,39 @@ class BufferOutput {
   Buffer::Instance& buffer_;
 };
 
+// Simple abstraction that provide a output string for streaming JSON output.
```

Comment:
Nit; add comments to this and BufferOutput that these implement the interface expected by StreamerBase.

Reply:
Sgtm.